pkg/storage/sync/storage.go

```diff
@@ -0,0 +1,165 @@
+package sync
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+	"github.com/weaveworks/ignite/pkg/client"
+	"github.com/weaveworks/ignite/pkg/storage"
+	"github.com/weaveworks/ignite/pkg/storage/watch"
+	"github.com/weaveworks/ignite/pkg/storage/watch/update"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type UpdateStream chan update.Update
+
+const updateBuffer = 4096 // How many updates to buffer
```

Comment:
document why you chose this value

Reply:
I just chose it by feeling :sweat_smile: but I'll add something